src/main/java/frc/robot/subsystems/ShooterTable.java

```diff
@@ -0,0 +1,13 @@
+package frc.robot.subsystems;
+
+public class ShooterTable {
+
+    public Double top,bottom,chute;
```

Comment:
These are member variables of a class. They should have the "m_" prefix.

Reply:
Not sure I agree, because they are intentionally public. This class is really like an old C struct. It just holds 3 pieces of data. So, code might look like "topSpeed = `lookupValue.top;". Having an "m_" is confusing for a public member.

Reply:
Agreed.